tests/publiccloud/nvidia.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright @ SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Opensource Nvidia test
+#          Drivers are only supported on SLE15SP4+
+# Maintainer: ybonatakis <ybonatakis@suse.com>
+
+use Mojo::Base 'consoletest';
+use registration;
+use testapi;
+use utils;
+use publiccloud::utils;
+
+sub run {
+    my ($self, $args) = @_;
+    $self->{provider} = $args->{my_provider};
+
+    script_run("hostname -f");
```

Comment:
is this necessary ?

Reply:
No really. leftover from the initial tests i was running. i will remove it